Newsletters: add 327 (2024-11-01)

harding opened the pull request:
- [x] Lede, releases/RCs, topic entries @harding 
- [x] New topics: duplex channels and DLEQ @harding 
- [x] Illustration for SuperScalar item @harding 
- [x] Notable merges @Gustavojfe

vostrnad commented on _posts/en/newsletters/2024-11-01-newsletter.md:
```diff
@@ -0,0 +1,144 @@
+  state updates compared to LN-Penalty's effectively unlimited number of
+  updates.  The intermediate duplex channels are used to allow
+  rebalances involving the LSP and two end users; these rebalances could
+  securely complete at offchain speeds, allowing user Alice to receive
```
Without Bob there's not much need for Alice:
```suggestion
  securely complete at offchain speeds, allowing a user to receive
```

vostrnad commented on _posts/en/newsletters/2024-11-01-newsletter.md:
```diff
@@ -0,0 +1,144 @@
+  inherent challenge in any channel factory design of managing
+  additional offchain state.  However, it has the advantage of being
+  something that could be implemented by a single team and made
+  interoperable with the standard LN without requiring many changes to
```
"Many" or "any"? If "many", what are the changes to the LN protocol that would be required?

harding replied:
Many is correct based on my conversation with ZmnSCPxj in https://bitcoinops.org/en/podcast/2024/10/31/ ; quick rough manual transcription:

49:34 we do need some changes at the protocol layer.  What I'm planning
to propose is that we have some kind of plugable factory concept.  We're
gonig to use an open v1 message to open this channel and then we're
going to have a TLV that says, "hey this is the protocol that we're
using and this is an identifier that specifies the particular protocol
that we're using, and then afterwards you can say "at the factory
layer we did some messaging and we need to change the funding output of
this channel".

vostrnad commented on _posts/en/newsletters/2024-11-01-newsletter.md:
```diff
@@ -0,0 +1,144 @@
+- **Draft BIP for DLEQ proofs:** Andrew Toth [posted][toth dleq] to the
+  Bitcoin-Dev mailing list a draft BIP and a link to an
+  [implementation][dleq imp] for generating and verifying proofs of
+  [discrete log equality][topic dleq] (DLEQ) for the elliptical curve used by
```
```suggestion
  [discrete log equality][topic dleq] (DLEQ) for the elliptic curve used by
```

bitschmidty reviewed (commented):
So far so good.

I am working to get the SuperScalar Podcast PR open today, targeting merge tomorrow. FYI

LarryRuane commented on _posts/en/newsletters/2024-11-01-newsletter.md:
```diff
@@ -0,0 +1,144 @@
+  vendor without waiting for large protocol changes that require
+  widespread agreement.  For example, a Lightning service provider (LSP)
+  that distributes wallet software could allow its users to more cheaply
+  open channels and receive inbound liquidity without sacrificing any of
```
```suggestion
  open channels and receive inbound liquidity without sacrificing
```

jonatack commented on _posts/en/newsletters/2024-11-01-newsletter.md:
```diff
@@ -0,0 +1,160 @@
+  [multi-signer coordination problem][news270 coordination].
+
+  Most of the leaves of the timeout tree are offchain funding
+  transactions for the common type of channel used today (LN-Penalty),
```
It doesn't look like "LN-Penalty" has been defined anywhere in this repo/website other than with introductory phrases like here, or "In the current LN channel design" in a previous newsletter.

It is also indirectly referred to in the Eltoo topic: "Although eltoo can be used with a penalty mechanism similar to the one used with existing LN channels, eltoo doesn’t need the penalty mechanism in order to be secure."

For anyone not familiar with LN-Penalty, maybe a link to a resource or topic could be good for this term.

jonatack reviewed (commented):
Looking good and the new topics are 👍

harding commented:
Rebased on master to pull in the deep dive podcast for a link to it.  Made edits (thanks @vostrnad @LarryRuane @jonatack @bitschmidty !), added lede, added topic text\*, added topic entries, removed image fixme (decided not to add one), and updated releases/RCs.  Reviewed @Gustavojfe section (thanks!).

\* There's quite a bit of last-minute text for the DLEQ/PODLE entry.  Sorry.  If anyone leaves a review here post-merge, I'll prioritize addressing that feedback Friday.

bitschmidty reviewed (approved):
Reviewed topics, lede, updates. Fixed up a couple minor items. Looks good!

Thank you @harding @Gustavojfe @LarryRuane @jonatack @vostrnad
